### PR TITLE
Implement Development Platform Marketplace auto discover

### PR DESCRIPTION
**Pull request by x4d3**
- Add MnoHelper.AutoConfigure and MnoHelper.AutoConfigure(host, path, key, secret)
- Refactor Connec.Client to force it to be instantiated with a group Id.
- Remove unused using
- Remove redundant full qualification

### CODE REVIEW

**Comment by x4d3** [comment]
@alachaum @BrunoChauvet Could you please review ?

File: src/Maestrano.Tests/app.config [hunk 1]
```diff
@@ -178,14 +179,24 @@
...
+
     <sso enabled="true" sloEnabled="true" idm="https://idm.myapp.com" idp="https://idp.sometenant.com" initPath="/sometenant/auth/saml/init.aspx" consumePath="/sometenant/auth/saml/consume" creationMode="virtual"/>
     <webhook>
     </webhook>
   </sometenant>
-  
-  
-  
+
+  <devPlatform
```
Comment (alachaum) [inline src/Maestrano.Tests/app.config]: The section name should be more specific as it is a general configuration file. e.g. maestranoDevPlatform

File: README.md [hunk 2]
```diff
@@ -1173,9 +1240,9 @@ The Maestrano API provides a built-in client - based on Restsharp - for connecti
 
 ```csharp
 // Pass the customer group id as argument
-var client = new Maestrano.Connec.Client("cld-f7f5g4");
+var client = new Maestrano.Connec.Client.New("cld-f7f5g4");
 // Using a specific Configuration Preset
-// var client = Maestrano.Connec.Client.With("somePreset").New("cld-f7f5g4");
+// var client = Maestrano.Connec.Client.New("cld-f7f5g4", "somePreset");
```
Comment (alachaum) [inline README.md:1249]: As we have breaking changes (preset syntax is different) we need to add migration notes.

**Review by alachaum — CHANGES_REQUESTED** [review]
See my two comments. The migration notes can be done once it's merged into 1.0.0. The other one (section group) should be changed I believe.

Also we should rename the branch 1.0.0 to 1.0 and create a alpha branch for the current master branch.

Otherwise, it looks much much better 😄

**Review by alachaum — APPROVED** [review]

**Comment by alachaum** [comment]
👍

**Pull request merged**